app/models/corporate_information_page.rb

```diff
@@ -33,11 +33,20 @@ def process_associations_before_save(new_edition)
   scope :accessible_documents_policy, -> { where(corporate_information_page_type_id: CorporateInformationPageType::AccessibleDocumentsPolicy.id) }
 
   def republish_owning_organisation_to_publishing_api
-    Whitehall::PublishingApi.republish_async(owning_organisation) if owning_organisation.present?
+    if state == "draft" && owning_organisation.is_a?(WorldwideOrganisation)
+      presenter = PublishingApi::WorldwideOrganisationPresenter.new(owning_organisation, state: "draft")
+      Services.publishing_api.put_content(presenter.content_id, presenter.content)
```

Comment:
It makes me confused that the path for dealing with the special case requires knowledge of the internals of the publishing API process, and manipulating content directly through those internals.

Whereas the "default" case has a neatly packaged method (`republish_async`).

Is there no way to use the more "typical" endpoints of the publishing mechanism? It could be that the answer is "no", or "not without significant refactoring", and that's fair, but I'm wary of coding ourselves into deeper and deeper exception paths and making any bugs harder to spot and diagnose.

Reply:
I agree 100%, and I don't like the fact that I'm depending on the institutional knowledge Bruce has of these publishing internals to know the solution to this problem. Unfortunately WorldwideOrganisations being non-editionable, but also having a `draft` state is an exception to the general rule of how whitehall content is published. 

Ultimately the solution would be to make WorldwideOrganisations editionable, and then draft editions can be published normally, however that's a significant piece of work and definitely outside the scope of this bugfix.

Reply:
Would you be willing to include this context in the commit message, to communicate the background to future devs?

Reply:
Sure, I'll amend it now :)